### ISSUE TITLE
Dataset operator pod goes into a crash backoff loop 

### ISSUE THREAD

**Issue by srikumar003** [datashim-io/datashim#180]
Deployment of dataset-operator results in error: 

```
Error: failed to start container "dataset-operator": Error response from daemon: failed to create shim: OCI runtime create failed: runc create failed: unable to start container process: exec: "/manager": stat /manager: no such file or directory: unknown
```
Root cause: The dataset operator and generate keys image haven't been updated on quay.io

**Comment by VassilisVassiliadis** [comment]
I think this means that the deployment is using an older image. The new image (after #177) contains the `manager` binary.

**Comment by srikumar003** [comment]
That's the cause. I am going to do a temporary revert of the manifest generation so that the older manifests work for the time being

**Comment by srikumar003** [comment]
Temporary fix (#181) merged with master. However, this will install images before #177 which are less compatible with k8s 1.22+. 

To get the latest operator, I advise manually building the operator image as below:

1. Clone datashim
```
$> git clone https://github.com/datashim-io/datashim.git
```
2. Prepare an alternate Docker registry for the operator image
3. Change build parameters
```
$> cd build-tools
```
Edit `Makefile` and replace values for `DOCKER_REGISTRY` to point to your alternate registry, and `ARCH` (`amd64` if you are on Intel)

Edit line 33: `@docker login -u="datashim+myrobot" -p=${DOCKER_TOKEN} quay.io ;\` to 
`@docker login -u=${DOCKER_USERNAME} -p=${DOCKER_TOKEN} ${DOCKER_REGISTRY} ;\ 
4. Run the commands
```
make build-components
make push-components
```
This should create the operator and generate-keys images in your chosen registry
5. Edit the manifests with the new images
```

```
